x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/sync_rules/connector_filtering_logic.tsx

```diff
@@ -260,7 +258,7 @@ export const ConnectorFilteringLogic = kea<
                 {
                   created_at: new Date().toISOString(),
                   field: '_',
-                  id: uuidv4(),
+                  id: 'DEFAULT',
```

Comment:
nit: I'm not always a huge fan of DRY, but this is a case where I would consider writing a small utility function to create this object so we don't have to maintain it twice.

Reply:
Ah, yeah, good point! Fixed :)